models/public/mobilenet-v3-large-1.0-224-tf/accuracy-check.yml

```diff
@@ -1,39 +1,11 @@
 models:
-  - name: mobilenet-v3-large-1.0-224-tf
-
-    launchers:
-      - framework: tf
```

Comment:
is it possible to add configuration for saved_model (obtained during preconvert script) and tf2 launcher of AC?

Reply:
Added